examples/byte-io/byte_io.bal

```diff
@@ -1,94 +1,48 @@
 import ballerina/io;
+import ballerina/log;
 
-// Reads a specified number of bytes from the given channel.
-function readBytes(io:ReadableByteChannel byteChannel,
-                   int numberOfBytes) returns (byte[], int) {
-    // Here is how the bytes are read from the channel.
-    var result = byteChannel.read(numberOfBytes);
-    match result {
-        (byte[], int) content => {
-            return content;
-        }
-        error readError => {
-            throw readError;
-        }
-    }
-}
-
-// Writes byte content with the given offset to a channel.
-function writeBytes(io:WritableByteChannel byteChannel,
-                    byte[] content,
-                    int startOffset = 0) returns int {
-    // Here is how the bytes are written to the channel.
-    var result = byteChannel.write(content, startOffset);
-    match result {
-        int numberOfBytesWritten => {
-            return numberOfBytesWritten;
-        }
-        error err => {
-            throw err;
-        }
+// Copies content from the source channel to a destination channel.
+function copy(io:ReadableByteChannel src,
+              io:WritableByteChannel dst) returns error? {
+    int readCount = 1;
+    byte[] readContent;
+    // Here is how to read all the content from
+    // the source and copy it to the destination.
+    while (readCount > 0) {
+        //Operation would attempt to read a maximum of 1000 bytes, the
```

Comment:
Change to "The operation attempts to read a maximum of 1000 bytes and returns with the available content, which could be < 1000."